Update PGDG rpm repo paths

hanefi opened the pull request:
Recently PGDG rpm repository paths changed. I made the necessary changes to the `rpm.sh` files

- Removed fedora from the supported OS list 
- Removed unused variables

To test the script you can use the following to use the scripts in this branch:
```
curl https://raw.githubusercontent.com/citusdata/packaging/update-pgdg-rpm-paths/community/rpm.sh | sudo bash
```

onurctirtir commented:
I wonder why we removed fedora from the scripts ?

edit: discussed in person and @hanefi noticed me that we don't even release packages for fedora for a long time

hanefi commented:
>I wonder why we removed fedora from the scripts ?

[PG Redhat Download page](https://www.postgresql.org/download/linux/redhat/) contains a note saying that not all Fedora packages are available. We ran into some issues in the past and dropped Fedora support starting from Citus 8.1.

```
The PostgreSQL Yum Repository currently supports:

Red Hat Enterprise Linux
CentOS
Scientific Linux
Oracle Linux
Fedora*
*Note: due to the shorter support cycle on Fedora, all supported versions of PostgreSQL are not available on this platform. We do not recommend using Fedora for server deployments.
```
